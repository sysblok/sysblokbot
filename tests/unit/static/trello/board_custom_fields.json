[
    {
      "id": "type_0",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "093946898f0a74bee9f218e55e31db722938563e140b0635704307912135a6bc",
      "display": {
        "cardFront": true
      },
      "name": "Автор",
      "pos": 16384,
      "type": "text"
    },
    {
      "id": "type_1",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "a91bc8b0b9b1ff076de078de356b1462beba9f4225c89986ef405c4371402b35",
      "display": {
        "cardFront": false
      },
      "name": "Google Doc",
      "pos": 32768,
      "type": "text"
    },
    {
      "id": "type_2",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "588488e8f09eb6dbaecbd89cf5fe83fc6f3299ca8a2323522410d96f80e7c00e",
      "display": {
        "cardFront": true
      },
      "name": "Редактор",
      "pos": 49152,
      "type": "text"
    },
    {
      "id": "type_3",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "9b23e842082fcaf352f26809eb8270a3f065d8d982a8af171428cac112e46b24",
      "display": {
        "cardFront": false
      },
      "name": "Название поста",
      "pos": 65536,
      "type": "text"
    },
    {
      "id": "type_4",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "92cc10f007fa367cd04b35929c26db94cc5d292247fdf04b575d0117c4282481",
      "display": {
        "cardFront": true
      },
      "name": "Иллюстратор",
      "pos": 81920,
      "type": "text"
    },
    {
      "id": "type_5",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "92cc10f007fa367cd04b35929c26db94cc5d292247fdf04b575d0117c4282481",
      "display": {
        "cardFront": true
      },
      "name": "Обложка",
      "pos": 81920,
      "type": "text"
    },
    {
      "id": "type_6",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "28c157040ba1375386ba5971564bfb17325043de973feae6128950036e5de20b",
      "display": {
        "cardFront": false
      },
      "name": "Сайт",
      "pos": 196608,
      "type": "checkbox"
    },
    {
      "id": "type_7",
      "idModel": "5e91d48a9e9d947b95fb0df7",
      "modelType": "board",
      "fieldGroup": "488f1b95d2449518badfd840b7a066d9b62debf1ccb4ec0805c68565320cde5a",
      "display": {
        "cardFront": false
      },
      "name": "Селектор",
      "pos": 212992,
      "options": [
        {
          "id": "5ede91f8c9525241e8c6785a",
          "idCustomField": "5ede91f8c9525241e8c67856",
          "value": {
            "text": "нет"
          },
          "color": "none",
          "pos": 69632
        }
      ],
      "type": "list"
    }
]
