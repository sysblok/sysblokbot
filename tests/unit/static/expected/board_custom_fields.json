[
    {
      "id": "type_0",
      "name": "Автор",
      "type": "text"
    },
    {
      "id": "type_1",
      "name": "Google Doc",
      "type": "text"
    },
    {
      "id": "type_2",
      "name": "Редактор",
      "type": "text"
    },
    {
      "id": "type_3",
      "name": "Название поста",
      "type": "text"
    },
    {
      "id": "type_4",
      "name": "Иллюстратор",
      "type": "text"
    },
    {
      "id": "type_5",
      "name": "Обложка",
      "type": "text"
    },
    {
      "id": "type_6",
      "name": "Сайт",
      "type": "checkbox"
    },
    {
      "id": "type_7",
      "name": "Селектор",
      "options": [
        {
          "id": "5ede91f8c9525241e8c6785a",
          "value": {
            "text": "нет"
          }
        }
      ],
      "type": "list"
    }
]
